examples/tests/sdk_tests/util/exec.test.w

```diff
@@ -0,0 +1,24 @@
+bring util;
+bring expect;
+
+test "exec() with valid program" {
+  let program = "echo";
+  let args = ["-n", "Hello, Wing!"];
+
+  let output = util.exec(program, args);
+  
+  expect.equal(output.stdout, "Hello, Wing!");
+  expect.equal(output.stderr, "");
+  expect.equal(output.status, 0);
+}
+
+test "exec() with invalid program" {
+  let program = "no-such-program";
+  let args = [""];
+
+  let output = util.exec(program, args);
```

Comment:
This should throw, no?

Reply:
Only the `shell()` API automatically throws on error. `exec()` lets the user decide what they want to do based on the command's output.